client/src/components/Intro/index.tsx

```diff
@@ -24,9 +28,16 @@ const Intro = ({
   pending,
   complete,
   completedChallengeCount,
-  slug
+  slug,
+  email,
+  onAlertClick
 }: IntroProps): JSX.Element => {
   const { t } = useTranslation();
+  const buttonVariation = (email: string | undefined): string => {
+    if (!email || emailToABVariant(email).isAVariant)
+      return t('buttons.donate');
```

Comment:
Does it matter that we will get more As than Bs?

Reply:
We should not and it does not matter. That block is only shown when user is signed. Also, If there are no emails it means the user is not signed it. We only report signed in users in GA saga.

Reply:
Great, thanks for explaining.